instrumentation-core/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/TracerAutoConfiguration.java

```diff
@@ -36,22 +36,19 @@
 @Configuration
 @EnableConfigurationProperties(TracerProperties.class)
 public class TracerAutoConfiguration {
-
-  @Autowired TracerProperties tracerProperties;
-
+  
   @Bean
-  public Tracer tracer() throws Exception {
+  @Autowired
+  public Tracer tracer(TracerProperties tracerProperties, SpanExporter spanExporter)
+      throws Exception {
     Tracer tracer = OpenTelemetry.getTracer(tracerProperties.getTracerName());
-    setLoggingExporter();
+    addSpanProcessor(spanExporter);
     return tracer;
   }
 
-  private void setLoggingExporter() {
-    if (!tracerProperties.isLoggingExporterEnabled()) {
-      return;
-    }
+  private void addSpanProcessor(SpanExporter spanExporter) {
+    SpanProcessor logProcessor = SimpleSpanProcessor.newBuilder(spanExporter).build();
```

Comment:
[Note] Is this intended to be used multiple times or just in tracer? 
Do we need any protections against repeated addition of a tracer or is that never an issue?